src/axes/timeAxis.ts

```diff
@@ -483,7 +482,7 @@ export module Axes {
       this._renderLabellessTickMarks(labelLessTicks);
 
       this._hideOverflowingTiers();
-      for (i = 0; i < tierConfigs.length; ++i) {
```

Comment:
I see. Previously it was redeclared due to a scoping conflict, so `let` resolves that.